Weblate: Update translations

photoprismci opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/photoprism/photoprism?pullRequest=2118) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/photoprism/photoprism?pullRequest=2118) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/photoprism/photoprism?pullRequest=2118) it.</sub>

weblate committed:
Added translation using Weblate (Turkish)

weblate committed:
Added translation using Weblate (Bulgarian)

weblate committed:
Translated using Weblate (Danish)

Currently translated at 6.7% (5 of 74 strings)

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/da/

weblate committed:
Added translation using Weblate (Romanian)

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/
